.github/release-image.yaml

```
@@ -0,0 +1,55 @@
+name: release
+
+on: 
+  workflow_dispatch: {}
+  push:
+    tags:
+      - '*'
```

I think it is  better to standardize with 'v' at the beginning like other civo project. What do you think? :thinking: 
```suggestion
      - 'v*.*.*'
```

---

yes that works